src/rai_toolbox/mushin/_compatibility.py

```diff
@@ -28,4 +28,4 @@ def _get_version(ver_str: str) -> Version:
     return Version(major=major, minor=minor, patch=int(patch_str))
 
 
-PL_VERSION: Final = _get_version(pytorch_lightning.__version__)
```

Comment:
This may break backwards compatibility with pytorch-lightning.  Looking at [this](https://github.com/Lightning-AI/lightning/blob/2415834aaf4f215ddb0d618bf415405d670f9114/src/pytorch_lightning/__init__.py#L9-L10) you should still be able to just access `pytorch_lightning.__version__`. I just did a fresh pip install and confirgmed:

```python
>>> import pytorch_lightning
>>> pytorch_lightning.__version__
'1.7.0'
```